apps/els_lsp/test/els_hover_SUITE.erl

```diff
@@ -490,13 +508,20 @@ nonexisting_type(Config) ->
     #{result := Result} = els_client:hover(Uri, 22, 15),
     %% The spec for `j' is shown instead of the type docs.
     Value =
-        case has_eep48_edoc() of
-            true ->
+        case list_to_integer(erlang:system_info(otp_release)) of
+            %% WIP: I think this might not be the way to go,
+            %% just putting it here for the PR.
```

Comment:
The results were different for me locally with OTP 24 and OTP 25.

Reply:
looking into this again, maybe it was something temporary

Reply:
I tried using the has_eep48_edoc() detector instead, but it is giving `true` on OTP 24 and OTP 25, but they get different results. This might be something we should maybe dive into and see why, and to fix it instead of futzing with the tests.

Reply:
"resolving" this thread to open one with my findings in the newest commit instead.